{"name": "testcaseadd3[【类型】输入为空，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一：和20.93开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\practice\\study\\study0824\\testcase\\test_add.py\", line 48, in testcaseadd3\n    result = self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1661497761889, "stop": 1661497761889}], "attachments": [{"name": "log", "source": "2d9b180f-8571-40b1-bf3b-6ecb78614ea0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "''"}, {"name": "y", "value": "20.93"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1661497761889, "stop": 1661497761890, "uuid": "9bdcbcf3-379a-4409-b720-508b5edabff3", "historyId": "18aa92624d4f60baf4508a56364d1586", "testCaseId": "2f8f937ca82ff540c8b5f46a0207716d", "fullName": "practice.study.study0824.testcase.test_add.Testadd#testcaseadd3", "labels": [{"name": "story", "value": "加法模块测试error"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "practice.study.study0824.testcase"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "Testadd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "33120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "practice.study.study0824.testcase.test_add"}]}